src/main/webapp/script.js

```diff
@@ -91,11 +91,28 @@ function fetchAuthentication() {
       createAuthenticationButton(
           user.authenticationUrl, 'btn-outline-success', 'Log Out', 'login');
 
-      // Show question submission box when logged in.
+      // Show submission forms when logged in.
       const questionSubmission = document.getElementById('post-question');
       if (questionSubmission) {
         questionSubmission.style.display = "block";
       }
+
+      const answerSubmission = document.getElementById('post-answer');
+      if (answerSubmission) {
+        answerSubmission.style.display = "block";
+      }
+
+      const commentSubmission = document.getElementsByClassName('post-comment');
+      if (commentSubmission != null) {
+        // The timeout is to wait for the dynamically generated forms of each
+        // answer to appear in the DOM.
+        setTimeout(() => {
+          for (element of commentSubmission) {
+            element.style.display = "block";
+          }
+        }, 500);
```

Comment:
Could you explain a little more about the timeout (for example the value 500 included at the end).

Reply:
done